Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/CommonComponents/AdditionalFieldsWindow.tsx

```diff
@@ -241,6 +241,7 @@ function AdditionalFieldsWindow(props: IProps): JSX.Element {
             cols={[
                 { key: 'FieldName', field: 'FieldName', label: 'Field Name', headerStyle: { width: 'auto' }, rowStyle: { width: 'auto' } },
                 { key: 'ExternalDB', field: 'ExternalDB', label: 'Ext DB', headerStyle: { width: 'auto' }, rowStyle: { width: 'auto' } },
+                { key: 'ExternalTable', field: 'ExternalTable', label: 'Ext Tbl', headerStyle: { width: 'auto' }, rowStyle: { width: 'auto' } },
```

Comment:
Does this label need to be shortened this aggressively? I think we can probably get away with `Ext Table`.

Reply:
Yeah probably not, changed so just ext is abbreviated